### ISSUE TITLE
Options gone

### ISSUE THREAD

**Issue by DerAustistYT** [leminlimez/Nugget#227]
After I updated my iPhone SE 2020 to iOS 18.1.1 every changes in the settings is not there anymore however the previous changes I did (as example charge limit to 85%) is still active. But the options to change those settings are gone..
Is that normal?

**Comment by SerStars** [comment]
Yes it is normal, but since you updated you can't revert or change the settings you applied unless you downgrade.
Nugget shows an unsupported text above since its newer than 18.2 developer beta 2/publix beta 1

**Comment by DerAustistYT** [comment]
> Yes it is normal, but since you updated you can't revert or change the settings you applied unless you downgrade.
> 
> Nugget shows an unsupported text above since its newer than 18.2 developer beta 2/publix beta 1

Well I found out, that this tool still works on 18.1.1 :) 
After I re ran the tool after the update, everything is back
